### PR TITLE
Remove setInterval implementations from camera controls

### PR DESCRIPTION
**Pull request by mlfarrell**
Improves responsiveness of initial zoom and mouse drag events.

Note that an engine-side fix is coming to address the mouse skipping/jumping situation.

### CODE REVIEW

File: src/clientSideScene/CameraControls.ts [hunk 1]
```diff
@@ -64,6 +64,27 @@ export type ReactCameraProperties =
...
+      now.getTime() - this.lastSend.getTime() > this.rateLimitMs
+    ) {
+      f()
+      this.lastSend = now
+    }
+  }
+
+  reset = () => {
+    this.lastSend = undefined
+  }
```
Comment (lf94) [inline src/clientSideScene/CameraControls.ts:85]: Everything's great, it just seems reset isnt called anywhere (I'm surprised tsc or lint passed this)

**Review by lf94 — APPROVED** [review]
Other than an unused reset everything looks good to me, feel free to merge, anything that improves camera is welcome asap :)

File: src/clientSideScene/CameraControls.ts (hunk 1 above)
Comment (mlfarrell) [reply src/clientSideScene/CameraControls.ts:85]: yea we may have wanted that later.  prob not now though.  feel free to remove it in a followup

File: ..env.development.local.swp
[the remark this answers is not in the document]
Comment (mlfarrell) [reply ..env.development.local.swp:1]: fml.  i didnt realize that was committed

**Pull request merged**